### DISCUSSION TITLE
답장으로 정보를 받아와 내보내기를 하려하는데

### DISCUSSION THREAD

**Discussion by moa12gg** [storycraft/node-kakao#745] [Q&A]
`CLIENT.on('chat', (data, channel) => {

  if(data.chat.type == 26 && data.text == '추방'){
    const sender = data.getSenderInfo(channel);
    const reply = data.attachment(); 
    const id = reply.src_userId;
    if(sender.userId == id){
        channel.sendChat('본인을 추방 할 수 없습니다.');
   }else{
     channel.kickUser(data.chat.id);
  }

 }`

channel.kickUser(data.chat.id); 이부분에서 오류가 나는데 어떤 게 문제인지 도움 주실 수 있을까요??
userId로 kickUser를 사용하려 했던게 문제일까요?

**Comment by artist012** [comment]
chat 안에 id는 없습니다만...

**Comment by benesiaxd** [comment]
총체적 난국
